cgp/hl_api.py

```diff
@@ -33,6 +35,10 @@ def evolve(
         `initialize_fitness_parents` and `step` method.
     min_fitness : float
         Minimum fitness at which the evolution is stopped.
+        Warning: This argument is deprecated and will be removed in the next
+        major release. Please use `termination_fitness` instead.
```

Comment:
i would not say major release. i think before 1.0 we can have this deprecation warning in place for one minor release, meaning we would remove this argument in 0.4.

Reply:
Good point, I changed it.